examples/applications/document-qa-chatbot/README.md

```diff
@@ -28,10 +28,31 @@ CREATE CUSTOM INDEX IF NOT EXISTS ann_index
   ON documents(embeddings_vector) USING 'StorageAttachedIndex';
 ```
 
-## Prepare the secrets file
+## Configure your OpenAI API Key
 
-As you would have already set up you should have a secrets.yaml file that contains the credentials to access the Vector Database
-and OpenAI APIs.
+Export some ENV variables in order to configure access to Open AI:
+
+```
+export OPENAI_ACCESS_KEY=...
+export OPENAI_URL=...
+export OPENAI_PROVIDER=azure
```

Comment:
shall we default to openai here instead of azure ?

Reply:
yes, I have updated it

Reply:
actully I have removed the whole application, we already have other applications, like the WebCrawler, that do the same things.

It is confusing to have too many examples that perform almost that same things